src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.Tests/ActivatorUtilitiesTests.cs

```diff
@@ -179,7 +180,7 @@ public void CreateInstance_ClassWithABC_MultipleCtorsWithSameLength_ThrowsAmbigu
         }
 
         [Fact]
-        public void CreateFactory_CreatesFactoryMethod()
+        public void CreateFactory_CreatesFactoryMethod_4Types_3Injected()
```

Comment:
This new test was added for coverage; the other new code paths already had coverage.